internal/cli/atlas/backup/compliancepolicy/enable_test.go

```diff
@@ -94,32 +96,3 @@ func TestEnableOpts_Run(t *testing.T) {
 	}
 	test.VerifyOutputTemplate(t, enableTemplate, expected)
 }
-func TestEnableOpts_Run_invalidEmail(t *testing.T) {
```

Comment:
[q] why did we remove these tests?

Reply:
Added back the email test, the other was a concern for the store and doesn't seem right to test it through the command. Let me know if you think otherwise though.